projects/ngx-prx-styleguide/src/lib/datepicker/datepicker.component.ts

```diff
@@ -79,7 +108,7 @@ export class DatepickerComponent implements AfterViewInit {
   }
 
   ngAfterViewInit() {
-    let options = {
+    let options: any = {
```

Comment:
I believe this should conform to `Pikaday.PikadayOptions`, and we have the Pikaday types installed already, so I'd suggest using those.

Reply:
Fixed